pytests/test_udct.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
```

Comment:
Write all tests using only the pylops method and not the methods of `ucurv`...  

Moreover, the bare minimum tests should have the dottest; in your case since you know the adjoint=inv, you can also add the assert that you have already.